vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -282,4 +283,90 @@ std::string Reddit::GetProfileImageUrl(const std::string& response) {
   return image_url;  // old reddit does not use account icons
 }
 
+void Reddit::OnMediaPublisherInfo(
+    const std::string& user_name,
+    ledger::PublisherInfoCallback callback,
+    ledger::Result result,
+    ledger::PublisherInfoPtr publisher_info) {
+  if (result != ledger::Result::LEDGER_OK &&
+      result != ledger::Result::NOT_FOUND) {
+    callback(ledger::Result::LEDGER_ERROR, nullptr);
+    return;
+  }
+  GURL url(REDDIT_USER_URL + user_name);
```

Comment:
1. should this url-encode the user_name? in case it has `/` or something in it
2. maybe add a check for url `is_valid()` or else `url.spec()` will throw if it's invalid

Reply:
1. Yes. Done.
2. Done.